integrations/tensorflow/iree_tf_compiler/TF/Passes.cpp

```diff
@@ -148,6 +152,16 @@ void buildTFImportPassPipeline(OpPassManager &pm) {
   pm.addPass(iree_compiler::Shape::createConvertHLOToShapePass());
 }
 
+
```

Comment:
```suggestion
```

Reply:
(from clang-format. Sticking it here so this can be merged from the UI without having to drop back down to the command lien)